library/kotlin/io/envoyproxy/envoymobile/Request.kt

```diff
@@ -0,0 +1,50 @@
+package io.envoyproxy.envoymobile
+
+import java.net.URL
+
+class Request internal constructor(
+    val method: RequestMethod,
+    val url: URL,
+    val headers: Map<String, List<String>>,
+    val trailers: Map<String, List<String>>,
+    val body: ByteArray?,
```

Comment:
https://github.com/lyft/envoy-mobile/pull/242#discussion_r302657799

Reply:
Resolving since the linked comment's decision is to discuss it further